test/unit/cmap/stream_description.test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const { StreamDescription } = require('../../../src/cmap/stream_description');
+const { expect } = require('chai');
+
+describe('StreamDescription - unit/cmap', function () {
+  describe('.new', function () {
+    context('when options are provided', function () {
+      context('when logicalSessionTimeoutMinutes is in the options', function () {
+        const options = { logicalSessionTimeoutMinutes: 5 };
+        const description = new StreamDescription('a:27017', options);
+
+        it('sets the property', function () {
+          expect(description.logicalSessionTimeoutMinutes).to.eq(5);
+        });
+      });
+
+      context('when logicalSessionTimeoutMinutes is not in the options', function () {
+        const description = new StreamDescription('a:27017', {});
+
+        it('sets logicalSessionTimeoutMinutes to undefined', function () {
+          expect(description.logicalSessionTimeoutMinutes).to.not.exist;
+        });
+      });
+
+      context('when loadBalanced is in the options', function () {
```

Comment:
should this be "and set to true"? and then should there be another test for "and set to false"?

Reply:
Improved tests to test for both true and false values.